ocpmodels/common/utils.py

```diff
@@ -319,17 +319,4 @@ def get_pbc_distances(pos, edge_index, cell, cell_offsets, neighbors, cutoff):
     # compute distances
     distances = distance_vectors.norm(dim=-1)
 
-    # remove zero distances
-    nonzero_idx = torch.nonzero(distances).flatten()
-    edge_index = edge_index[:, nonzero_idx]
-    distances = distances[nonzero_idx]
```

Comment:
Not sure if you did this already, but have you checked to make sure distances are never zero? Otherwise it'll cause things to break in the energy-gradient-based force models. In theory, they should never be zero..

Reply:
In ML-driven relaxations atoms could be pushing against each other, so might still be good to leave those lines to remove zero-distance edges in there. Thoughts?